### ISSUE TITLE
[error]failed to parse title: invalid semver

### ISSUE THREAD

**Issue by peterbe** [ahmadnassri/action-dependabot-auto-merge#16]
https://github.com/mdn/yari/pull/1231 failed because of this error:
<img width="623" alt="Screen Shot 2020-09-10 at 7 12 16 AM" src="https://user-images.githubusercontent.com/26739/92722263-4edacd80-f335-11ea-8df3-ea5ef33fb82c.png">
https://github.com/mdn/yari/pull/1231/checks?check_run_id=1094792761

[Workflow here](https://github.com/mdn/yari/blob/master/.github/workflows/auto-merge.yml)

**Comment by ahmadnassri** [comment]
"technically" this is correct behavior...

GitHub Actions uses the `@*` to identify versions using tags & branches ... 

an action could point to `@master` for example, and dependabot is pushing PRs seemingly by "recency" rather than version increment...

ignoring for a second that this action is triggered on a pull request to update the action itself ... because it's getting too meta! 

in this auto-merge action, `v1` _(which is the tagged version of an action)_ is not a valid semver, and therefore the logic to compare versions is not going to work, so it "soft fails" (all checks are green) and falls back to human intervention ...

I could squint my eyes and see a possibility of attempting to "fix" the version (`v1` to `v1.*.*`) before comparing it...

but first I wanted to explain the workflow and show that's working as expected here without causing failures

**Comment by peterbe** [comment]
Oh I see. It's a bit tricky because the "unexpected error" is being used as a form of "safe failure". Ideal would be a warning instead "##[warning] Incompatible semver versions. Can't compare, so needs human eyes" :)

This notation of `uses: technote-space/get-diff-action@v3` appears to be very common on GitHub Actions but very rare on things like NPM or PyPI packages. 

I'll leave it to you to "squint your eyes". But for now, I'll just merge mine :)

**Comment by ahmadnassri** [comment]
- made the change to use warnings: https://github.com/ahmadnassri/action-dependabot-auto-merge/commit/5bab4aa8c15c7c4633dde7428cb3518b56ed2e60
- yeah that notation was a pain to even get setup properly in releases .. see: https://github.com/ahmadnassri/action-dependabot-auto-merge/blob/5bab4aa8c15c7c4633dde7428cb3518b56ed2e60/.github/workflows/release.yml#L48-L69

**Comment by ahmadnassri** [comment]
as for your merging that change, you can also just set it to `v2` and ignore the minor/patch version specific PRs from dependabot for actions ....

unless you want those ...

**Comment by peterbe** [comment]
I am so sorry for being a pain but...

I could understand how it could error on `v1  -> v2.0.1` or something because `1` isn't a valid semver. 
But what about this one:
```
title: "Bump ahmadnassri/action-dependabot-auto-merge from v2.0.1 to v2.0.4"
depName: ahmadnassri/action-dependabot-auto-merge
##[warning]failed to parse title: invalid semver
```
As seen here: https://github.com/mdn/yari/pull/1243/checks?check_run_id=1100241207

So meta! It's [a PR to upgrade your/this action from 2.0.1 to 2.0.4](https://github.com/mdn/yari/pull/1243).

**Comment by ahmadnassri** [comment]
so very meta!

`2.0.1` had a bug, and was fixed in `v2.0.2` which addresses this semver parsing error

upgrade to latest (v2.0.4) or simply use v2 in your workflow: ahmadnassri/action-dependabot-auto-merge@v2 to use the latest within the same major

**Comment by peterbe** [comment]
Still a bug in 2.0.4. 
See https://github.com/mdn/yari/pull/1326/checks?check_run_id=1175038089

```
using workflow's "target": 
- match:
    dependency_type: all
    update_type: 'semver:minor'

title: "Bump ahmadnassri/action-dependabot-auto-merge from v2.0.4 to v2.1.1"
depName: ahmadnassri/action-dependabot-auto-merge
Warning: failed to parse title: invalid semver
```

It failed to upgrade from `v2.0.4` to `v2.1.1`. Is it because of the little `v` prefix?
